eth/abc.py

```diff
@@ -631,10 +631,33 @@ def persist_block(self,
             as genesis. Providing a ``genesis_parent_hash`` allows storage of blocks that
             aren't (yet) connected back to the true genesis header.
 
-        Assumes all block transactions have been persisted already.
+        WARNING: This API assumes all block transactions have been persisted already. Use
```

Comment:
I think that syntactically for RST this should be `.. warning: This API assumes...`